File: word_load_into_tries.py
f = open('words.txt')
words = f.read().split('\n')
f.close()


f = open('web_modules/res/js/trie_words.js', 'w')
for word in words:
	f.write('trie.addWord(\'' + word + '\');\n')
f.close()

File: word_load_into_tries.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main (){
	string dataline;
	ifstream infile;
	infile.open("words.txt");
	ofstream myfile;
	myfile.open ("web_modules/res/js/trie_words.js");
	while(!infile.eof()){
		getline(infile, dataline);
		cout << dataline;
		myfile << "trie.addWord('" << dataline << "');\n";
	}
	infile.close();
	myfile.close();
	return 0;
}